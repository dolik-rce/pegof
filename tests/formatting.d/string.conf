input formatting.d/string.peg
