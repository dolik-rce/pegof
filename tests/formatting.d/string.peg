X <- "ABC" '' "" 'DEF' "\x1b\u005f\u1E9E"
