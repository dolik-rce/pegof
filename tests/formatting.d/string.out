X <- "ABC" "" "" "DEF" "\x1b_\u1e9e"
